src/ASM/ASMunstruct.h

```diff
@@ -134,6 +134,12 @@ class ASMunstruct : public ASMbase
   static void Sort(int u, int v, int orient,
                    std::vector<LR::Basisfunction*>& functions);
 
+  //! \brief Return all boundary functions that are covered by the given set of input nodes
+  //! \param   nodes List of (0-indexed) patch local node IDs
+  //! \returns       Node IDs (0-indexed) for boundary functions whos support is
+  //                 completely covered by the union of the support in the input array
+  virtual IntVec getBoundaryNodesCovered(const IntVec& nodes) const ;
```

Comment:
fjern space før ; og det holder med en space etter `\param` og `\return` i doxyen, og siden `nodes` kun er input, bruk `\param[in]`. One spell error too (whos --> whose).